include/sealib/compactarray.h

```diff
@@ -0,0 +1,48 @@
+#ifndef SEALIB_COMPACTARRAY_H_
+#define SEALIB_COMPACTARRAY_H_
+
+#include <math.h>
+#include <third-party/boost/include/boost/dynamic_bitset.hpp>
+
+typedef boost::dynamic_bitset<> Group;
+
+/**
+ * This compact array stores a given number of values in as low space as
+ * possible.
+ * Therefore an "epsilon" parameter is used: the smaller epsilon is, the larger
+ * the storage groups will be.
```

Comment:
Please explain "storage" and how epsilon influence the size of the storage.

Reply:
I think I can delete this sentence. Epsilon does not affect the total space required, and mentioning it is actually an unnecessary detail.